### DISCUSSION TITLE
JavaScript autocomplete not working

### DISCUSSION THREAD

**Discussion by Error-Cookie** [coder/code-server#3923] [Q&A]
I want to write javascript code but ctrl+space does not show autocomplete suggestions and intellisense also is not working. How can I enable this? I have enabled everything in the settings.

**Comment by jsjoeio** [comment]
Hmm autocomplete looks to be working for me in 3.11.1. Do you mind providing steps to reproduce your issue? Or sharing a video?

![image](https://user-images.githubusercontent.com/3806031/128915561-8c152be3-62bc-47b0-814c-aec3e1bc5f68.png)
